hack/verify-golint.sh

```diff
@@ -24,9 +24,13 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::verify_go_version
 
 if ! which golint > /dev/null; then
```

Comment:
seems like we should always use the vendored one to match ci? 🤔 

(I forget if k/k uses this pattern elsewhere, but always using the copy from vendor has worked well for other projects at least...)

Reply:
make sense

Reply:
update the PR title? :-)